sql/migrations/20200409171600_world.sql

```diff
@@ -0,0 +1,61 @@
+DROP PROCEDURE IF EXISTS add_migration;
+delimiter ??
+CREATE PROCEDURE `add_migration`()
+BEGIN
+DECLARE v INT DEFAULT 1;
+SET v = (SELECT COUNT(*) FROM `migrations` WHERE `id`='20200409171600');
+IF v=0 THEN
+INSERT INTO `migrations` VALUES ('20200409171600');
+-- Add your query below.
+
+
+--remove all incorrect trainers from An Imps Request (8419) Richard Kerwin (4565), Luther Pickman (4564), Ursula Deline (5495), Sandahl (5496), Thistleheart (5171), Alexander Calder (5173), Grol'dar (3324), Mirket (3325) and Kartosh (988)
+DELETE FROM mangos.creature_questrelation WHERE quest = 8419 AND id NOT IN(461,3326,4563);

```

Comment:
don't need database ref here

Reply:
Cheers for the review, I have removed all database references.